Automattic/sensei
# Add "Publish your first course" task

Fixes #5639

### Changes proposed in this Pull Request

* Add Publish your First Course task;
* Add tests for the task added;
* Register task on the provider;

### Testing instructions

On a WP installation containing Sensei LMS on this branch:

1.  Run `curl --user "user:application password" http://your-host/wp-json/sensei-internal/v1/home` and make sure that the task "Publish your first Course" has `done` set to `false`, with the URL pointing to create a new course;
2.    Create a course/import the sample course, and verify if that task still has `done` set to `false`, with the URL still pointing to create a new course.
3. Publish the sample course, and verify if that task still has `done` set to `false`, with the URL still pointing to create a new course;
4. Now, create another course, and verify if that task still has `done` set to `false`, but with the URL now pointing to the edit page for that course you just created;
5. Publish the course you just created, and verify if that task now has `done` set to `true`, and the URL is now `null`;
